<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widgetAddApp" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>550</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>550</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget #widgetAddApp {
	background-color:  rgb(246, 235, 97);
}</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>720</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 51, 102);
color: white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Audit &amp; Login Logs</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="logWidget">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>80</y>
      <width>650</width>
      <height>410</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
    border: 1px solid  rgb(0, 51, 102);
    border-top: none;
    background-color: white;
}

QTabBar::tab {
    font-family: &quot;Arial&quot;;
    font-size: 14px;
    font-weight: bold;
    color: #555;
    background: white;
    border: 1px solid #d3d3d3;
    border-bottom-color: #d3d3d3;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 100px;
    max-width: 200px;
    padding: 6px 12px;
    margin-right: 2px;
	height: 20%;
}

QTabBar::tab:hover {
    background: #e0e0e0;
    color: #333;
}

QTabBar::tab:selected {
    background: rgb(0, 51, 102);
    color: white;
    border-bottom-color: white;
    min-width: 120px;
}

QTabBar::tab:focus {
    border: 1px solid #4a90e2;
    background-color:  rgb(0, 51, 102);
}

QTabBar::tab:selected:focus {
    border-bottom: 2px solid #4a90e2;
	background-color:  rgb(0, 51, 102);
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabAudit">
     <attribute name="title">
      <string>Audit Logs</string>
     </attribute>
     <widget class="QTableWidget" name="tableAudit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>648</width>
        <height>375</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    border: 1px solid  rgb(0, 51, 102);
    background-color: white;
    gridline-color: #dcdcdc;
}

QHeaderView::section {
    background-color: rgb(0, 51, 102);
    color: white;
    padding: 2px;
}

QTableWidget::item {
    font-size: 10px;
	padding-left: 2px;
}
</string>
      </property>
      <column>
       <property name="text">
        <string>User ID</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Action</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Timestamp</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLogin">
     <attribute name="title">
      <string>Login Logs</string>
     </attribute>
     <widget class="QTableWidget" name="tableLogin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>648</width>
        <height>375</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    border: 1px solid  rgb(0, 51, 102);
    background-color: white;
    gridline-color: #dcdcdc;
}

QHeaderView::section {
    background-color: rgb(0, 51, 102);
    color: white;
    padding: 2px;
}

QTableWidget::item {
    font-size: 10px;
	padding-left: 2px;
}
</string>
      </property>
      <column>
       <property name="text">
        <string>User ID</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Action</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP Address</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Timestamp</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushBtnVOL">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton[active=&quot;true&quot;] {
	background-color: rgb(246, 235, 97);
	color:  rgb(0, 51, 102);
}

QPushButton {
	background-color: rgb(0, 51, 102);
	color: white;
	border-radius: 10px;
}

QPushButton:hover {
	background-color: #2980b9;
}
</string>
    </property>
    <property name="text">
     <string> View Old Login Logs</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnVAL">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton[active=&quot;true&quot;] {
	background-color: rgb(246, 235, 97);
	color:  rgb(0, 51, 102);
}

QPushButton {
	background-color: rgb(0, 51, 102);
	color: white;
	border-radius: 10px;
}

QPushButton:hover {
	background-color: #2980b9;
}
</string>
    </property>
    <property name="text">
     <string> View Old Audit Logs</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
